Resources/Prototypes/Catalog/Fills/Crates/antag.yml

```diff
@@ -0,0 +1,15 @@
+- type: entity
+  id: CratePirateChest
+  suffix: antag
+  parent: CratePirate
+  components:
+  - type: StorageFill
+    contents:
+      - id: HatBandRed
+        amount: 1
+      - id: Cutlass
+        amount: 1
+      - id: IngotGold
```

Comment:
starting with gold seems kind of strange, considering that its just free money for the pirates

Reply:
good point, not exactly sure what could be in a pirate chest